stigs/system/hsp-audit-stig-ubuntu-20-010165-chcon.yaml

```
@@ -0,0 +1,17 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit: 
+# https://www.accuknox.com/kubearmor/ 
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorHostPolicy
+metadata:
+  name: hsp-audit-stig-ubuntu-20-010165-chcon
+spec:
+  severity: 5 
+  tags: ["STIG", "Ubuntu", "UBTU-20-010165", "chcon"] 
+  message: "Alert! chcon binary execution detected" 
+  process:
+    matchPaths:
+    - path: usr/bin/chcon
```

```suggestion
  process:
    matchPaths:
    - path: /usr/bin/chcon
```

Wondering why does a SELinux binary rule is in a Ubuntu STIG.